sdk/invoke.js

```diff
@@ -21,10 +21,25 @@ const invoke = async function invoke(funcName, apiKey, funcData) {
     resolveWithFullResponse: true,
   };
   logger.debug('Invoking function', options);
-  const res = await rp.post(options);
-  const { statusCode, headers, body } = res;
-  logger.debug('Invoke response', { statusCode, headers, body });
-  return res;
+  try {
+    const res = await rp.post(options);
+    const { statusCode, headers, body } = res;
+    logger.debug('Invoke response', { statusCode, headers, body });
+    return res;
+  } catch (err) {
+    let parsedError;
+    try {
+      parsedError = JSON.parse(err.error);
+    } catch (nestedErr) {
+      throw new Error(err.error || err);
+    }
+    if (parsedError.stack && (parsedError.error || parsedError.message)) {
```

Comment:
consider naming `(parsedError.error || parsedError.message)` as it's used twice

Reply:
Will do.